### ISSUE TITLE
Add the Unlicense or CC0

### ISSUE THREAD

**Issue by RCW679** [CodaFi/C-Macs#3]
as title

**Comment by antirez** [comment]
Yep. No license = it can not be used. MIT is probably what you want. No responsibility on your side and freedom for others to do what they want.
